elide-core/src/main/java/com/yahoo/elide/utils/coerce/converters/ElideTypeConvertor.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.utils.coerce.converters;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface ElideTypeConvertor {
```

Comment:
I think the correct spelling is converter

Reply:
Noted, will correct the typo.